CRM/Remoteevent/Registration.php

```diff
@@ -749,9 +749,32 @@ public static function registerAdditionalParticipants(RegistrationEvent $registr
         $profile = CRM_Remoteevent_RegistrationProfile::getRegistrationProfile(
            $event['event_remote_registration.remote_registration_additional_participants_profile']
         );
-        foreach ($additionalParticipants as $additionalParticipantNo => &$additionalParticipant) {
+
+        $additionalContactsData = [];
```

Comment:
`s/$additionalContactsData/$additionalParticipantsContactData` for clarity that this is about the additional participants.